assets/wizards/popups/views/campaign-group-management/index.js

```diff
@@ -0,0 +1,177 @@
+import { __ } from '@wordpress/i18n';
+import { ESCAPE } from '@wordpress/keycodes';
+
+/**
+ * Material UI dependencies.
+ */
+import DeleteIcon from '@material-ui/icons/Delete';
+
+/**
+ * Internal dependencies
+ */
+import {
+	withWizardScreen,
+	ActionCard,
+	Button,
+	Notice,
+	Popover,
+	TextControl,
+} from '../../../../components/src';
+import './style.scss';
+
+/**
+ * Popup group screen.
+ */
+const CampaignGroupManagement = ( { wizardApiFetch } ) => {
+	const [ groups, setGroups ] = useState( null );
+	const [ inFlight, setInFlight ] = useState( false );
+	const [ newGroupName, setNewGroupName ] = useState( '' );
+	const [ addNewPopoverIsVisible, setAddNewPopoverIsVisible ] = useState( false );
+	const [ error, setError ] = useState( null );
+
+	const createTerm = term => {
+		setInFlight( true );
+		setError( false );
+		wizardApiFetch( {
+			path: '/wp/v2/newspack_popups_taxonomy',
+			method: 'POST',
+			quiet: true,
+			data: {
+				name: term,
+				slug: term,
+			},
+		} )
+			.then( () => {
+				setInFlight( false );
+				setNewGroupName( '' );
+				retrieveTerms();
+			} )
+			.catch( e => {
+				const message =
+					e.message || __( 'An error occurred when creating this group.', 'newspack' );
+				setError( message );
+				setInFlight( false );
+			} );
+	};
+
+	const deleteTerm = id => {
+		setInFlight( true );
+		setError( false );
+		wizardApiFetch( {
+			path: `/wp/v2/newspack_popups_taxonomy/${ id }?force=true`,
+			method: 'DELETE',
+			quiet: true,
+		} )
+			.then( () => {
+				setInFlight( false );
+				retrieveTerms();
+			} )
+			.catch( e => {
+				const message =
+					e.message || __( 'An error occurred when deleting this group.', 'newspack' );
+				setError( message );
+				setInFlight( false );
+			} );
+	};
+
+	const retrieveTerms = () => {
+		setInFlight( true );
+		setError( false );
+		wizardApiFetch( {
+			path: '/wp/v2/newspack_popups_taxonomy?_fields=id,name,count',
+			quiet: null !== groups,
+		} )
+			.then( terms => {
+				setInFlight( false );
+				setGroups( terms );
+			} )
+			.catch( e => {
+				const message = e.message || __( 'An error occurred when retrieving groups.', 'newspack' );
+				setError( message );
+				setInFlight( false );
+			} );
+	};
+
+	useEffect( retrieveTerms, [] );
+
+	return (
+		<Fragment>
+			{ error && <Notice noticeText={ error } isError /> }
+			<div noBorder className="newspack-campaigns-wizard-groups__add-ui">
+				<Button
+					isPrimary
+					isSmall
+					onClick={ () => setAddNewPopoverIsVisible( true ) }
+					disabled={ !! inFlight }
+				>
+					{ __( 'Add New', 'newspack' ) }
+				</Button>
+				{ addNewPopoverIsVisible && (
+					<div className="newspack-campaigns-wizard-groups__add-new-button">
+						<Popover
+							position="bottom left"
+							onFocusOutside={ () => setAddNewPopoverIsVisible( false ) }
+							onKeyDown={ event => ESCAPE === event.keyCode && setAddNewPopoverIsVisible( false ) }
+						>
+							<MenuItem
```

Comment:
What's this `MenuItem` here for? I don't see the button and lint is complaining that `setPreviewPopoverIsVisible` isn't a real function.

Reply:
The `MenuItem` is a hidden element to provide a way for screen reader users to close the popover. This exists in the other popovers we use in the Campaigns tab as well.

Reply:
Got it, then the only change is the click event should be `setAddNewPopoverIsVisible( false )`

Reply:
Whoops, fixed in 5fbb5d3.